src/note/note.entity.ts

```diff
@@ -0,0 +1,18 @@
+import { Entity, Column, PrimaryGeneratedColumn } from 'typeorm';
+@Entity()
+
+export class notes {
```

Comment:
nazwa pliku wskazuje, ze klasa powinna nazywać się NoteEntity

Reply:
brakuje mi w całości zasad walidacji bazodanowych, np
```
  @Column({ type: 'varchar', length: 10, nullable: false })
  title: string;
```

bardzo ważne, żeby te zasady były spójne z walidacją jaką zróbisz w swoim DTO, czyli np jeśli tutaj ustawisz, że pole w bazie ma mieć length: 255 to w Twoim NoteDTO max length powinien być na @MaxLength(255), jeśli dasz nullable true no to @Required(), jak nullable: false to @Optional() itd

Reply:
Poprawione